pkg/quota/v1/resources_test.go

```diff
@@ -319,3 +320,93 @@ func TestIsNegative(t *testing.T) {
 		}
 	}
 }
+
+func TestIntersection(t *testing.T) {
```

Comment:
is it worthwhile to test the new behavior in CalculateUsage?

Reply:
it's tested in `TestSyncResourceQuota`